charts/seed-controlplane/charts/csi-alicloud/templates/csi-diskplugin-secret.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```

Comment:
Can't you re-use the secret with name `cloudprovider`? It already has all the information you require?

Reply:
Yes. Thanks for your suggestion.